src/geometry.jl

```diff
@@ -903,6 +903,92 @@ function lifting_line_geometry!(r, c, grids, xc=0.25)
     return r, c
 end
 
+"""
+    lifting_line_geometry(surfaces, xc=0.25)
+Construct a lifting line representation of the surfaces in `grids` at the
+normalized chordwise location `xc`.  Return the lifting line coordinates
+and chord lengths.
+# Arguments
+ - `surfaces`: Vector of the surfaces to create lifting line from.
+ - `xc`: Normalized chordwise location of the lifting line from the leading edge.
+    Defaults to the quarter chord
+# Return Arguments:
+ - `r`: Vector with length equal to the number of surfaces, with each element
+    being a matrix with size (3, ns+1) which contains the x, y, and z coordinates
+    of the resulting lifting line coordinates
+ - `c`: Vector with length equal to the number of surfaces, with each element
+    being a vector of length `ns+1` which contains the chord lengths at each
+    lifting line coordinate.
+"""
+function lifting_line_geometry(surfaces::Vector{Matrix{SurfacePanel{TF}}}, xc=0.25) where TF
```

Comment:
probably best to overload this for a `system::System` object rather than `surfaces`